src/test/obj_debug/Makefile

```diff
@@ -39,6 +39,7 @@ OBJS = obj_debug.o
 LIBPMEM=y
 LIBPMEMOBJ=y
 
+BUILD_STATIC_DEBUG=n
```

Comment:
Please add comment why you do this.

Reply:
Maybe add BUILD_STATIC variable just as an alias for BUILD_STATIC_DEBUG and BUILD_STATIC_NONDEBUG?